hlapipeline/utils/astrometric_utils.py

```diff
@@ -134,6 +134,9 @@ def create_astrometric_catalog(inputs, **pars):
         Specify whether or not to only use sources from GAIA in output catalog
         Default: False
 
+    existing_wcs : HST.wcs object
+        existing WCS object specified by the user
+
```

Comment:
These changes will need to be merged in or compared to the changes implemented in #37.

Reply:
changes will be implemented by PR #37